charts/alfresco-adf-app/values.yaml

```diff
@@ -23,9 +23,16 @@ serviceAccount:
 podAnnotations: {}
 podLabels: {}
 
-podSecurityContext: {}
-
-securityContext: {}
+podSecurityContext:
+  runAsUser: 101
+  runAsNonRoot: true
+
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - NET_RAW
+      - ALL
```

Comment:
Is `NET_RAW` not a part of `ALL`?

Reply:
good catch, I simply copy-pasted what was in our common library, will create a task